src/transformers/Twitch.js

```diff
@@ -0,0 +1,45 @@
+import { URL } from 'url';
+import { includesSomeOfArray } from './utils';
+
+const ignoredPaths = ['/followers', '/settings', '/videos'];
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+
+  return (
+    [
+      'player.twitch.tv',
+      'www.twitch.tv',
+      'twitch.tv',
+      'clips.twitch.tv',
+    ].includes(host) &&
+    !includesSomeOfArray(pathname, ['followers', 'settings']) &&
```

Comment:
You should do this in a more generic way.
I can add some extra tests so this will fail and you'll have to update the code each time and that's not what we want.

I think you should focus on what _is_ allowed in this function, instead of what's not.

Reply:
In this case should I just check if it includes host, twitch.tv, player.twitch, clips.twitch and if the url includes any of: `clips`, `videos`, `collections`? I guess that could be a better way to do this, but the url `https://twitch.tv/jlengstorf/videos` will be whitelisted so I'm not sure how this should look like. 

I've used the not includesSomeOfArray mostly because another transformer already followed this pattern.

Reply:
I think you should check if it includes `videos` _and_ the `pathname` has 3 parts